### DISCUSSION TITLE
Whitebeet Status Messages

### DISCUSSION THREAD

**Discussion by Bobbybigears** [Sevenstax/FreeV2G#105] [General]
Hi,

In revision 2.14 of the White Beet User Manual, in Section 15 (Control Pilot Service) and specifically the section on Status Messages (15.3), it says that 'Status Messages will be sent asynchronously, without being explicitly requested. They will be sent using the sub ID 0xFF. The data will consist of a single byte containing the status code followed by additional data'.
Underneath this paragraph there are a list of Status Messages that have the sub IDs 0x80 and 0x81. Are these supposed to be the status codes or are they separate Status Messages? If so, is there a list of the Status Codes returned by the 0xFF Status Message?

Many thanks, Rob.

**Comment by jpo-stx** [comment]
Hello Rob,

the text

>They will be sent using the sub ID 0xFF

is wrong. Status messages are sent with the **request ID** 0xFF.

So you will habe a status message with sub ID 0x80 with request ID 0xFF and a status message with sub ID 0x81 and request ID 0xFF.
